<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirlineServices</class>
 <widget class="QMainWindow" name="AirlineServices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionScheduleView"/>
    <addaction name="actionShowPriceTickets"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScheduleView">
   <property name="text">
    <string>ScheduleView</string>
   </property>
  </action>
  <action name="actionShowPriceTickets">
   <property name="text">
    <string>ShowPriceTickets</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
